platform/docs/src/css/custom.css

```
@@ -348,31 +207,40 @@ body {
+.markdown h2,
+.markdown h3,
+.markdown h4,
+.markdown h5,
+.theme-doc-markdown h1,
+.theme-doc-markdown h2,
+.theme-doc-markdown h3,
+.theme-doc-markdown h4,
+.theme-doc-markdown h5 {
   color: var(--ifm-color-primary);
   font-weight: 400;
   font-family: 'Inter', sans-serif;
```

Stylelint (17.13.0)

[error] Expected no quotes around "Inter" (font-family-name-quotes)

(font-family-name-quotes)